fossunited/api/tickets.py

```
@@ -245,36 +245,73 @@ def get_percentage_change(today: float, yesterday: float) -> float:
 
 
 @frappe.whitelist()
-def get_sold_tickets(event_id: str, filters: dict = {}) -> list:
+def get_tickets_with_custom_fields(event_id: str, filters: dict = {}) -> list[dict]:
```

Ruff (0.14.7)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)